plugins/c9.cli.open/open.js

```diff
@@ -25,19 +27,28 @@ define(function(require, exports, module) {
             cmd.addCommand({
                 name: "open", 
                 info: "     Opens a file or directory.",
-                usage: "[--wait] <path>",
+                usage: "[--wait] [--pipe] <path>",
                 options: {
                     "wait": {
                         description: "Wait until the file(s) are closed",
                         "default": false,
                         "boolean": true
+                    },
+                    "pipe": {
+                        description: "Pipe data from a command into c9",
+                        "default": false,
+                        "boolean": true
                     }
                 },
                 check: function(argv) {
-                    if (argv._.length < 2 && !argv["path"])
+                    if (argv._.length < 2 && !argv["path"] && !argv.pipe)
                         throw new Error("Missing path");
                 },
                 exec: function(argv) {
+                    if(argv.pipe) {
```

Comment:
a small formatting error, we really need to get auto-formatting working properly in the ide:)